postgres/datadog_checks/postgres/util.py

```diff
@@ -1127,3 +1127,32 @@ def trim_leading_set_stmts(sql):
         {'name': 'locks.idle_in_transaction_age', 'type': 'gauge'},
     ],
 }
+
+# Column statistics from pg_statistic
+# Requires SELECT permissions on specific pg_statistic columns:
+#   `GRANT SELECT(starelid, staattnum, stawidth, stadistinct) ON pg_statistic TO datadog;`
+COLUMN_METRICS = {
+    'descriptors': [('schemaname', 'schema'), ('tablename', 'table'), ('attname', 'column')],
+    'metrics': {
+        'avg_width': ('column.avg_width', AgentCheck.gauge),
+        'n_distinct': ('column.n_distinct', AgentCheck.gauge),
+    },
+    'query': """
+SELECT
```

Comment:
Lets' add `null_frac` as well.

Reply:
Sounds good! I've had a few different chats this week around collecting + posting them in a way that matches our use-cases + data infra; going to send a doc to you with updated thoughts shortly!